infra/modules/notifications-app/resources/main.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_pinpoint_app" "app" {
+  name = var.name
+}
+
+resource "aws_pinpoint_email_channel" "app" {
```

Comment:
nit: can we move this resource to its own file? something like:
- `email.tf`
- `email-notifications.tf`
- `email-channel.tf`

we eventually also want to add an SMS channel so might as well organize things now